codebuild/common-posix.sh

```diff
@@ -6,5 +6,10 @@ mkdir /tmp/build
 cd /tmp/build
 
 cmake -DPERFORM_HEADER_CHECK=ON -DENABLE_SANITIZERS=ON $@ $CODEBUILD_SRC_DIR
-make
+make -j 12
```

Comment:
trivial: maybe just `-j` without a number and let make pick the right # for the machine?